app/lib/ui/components/buttons/component_elevated_button.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
```

Comment:
rename the file removing component to_ have elevated_button.dart

Reply:
done